website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -250,3 +250,156 @@ The GET endpoints above can optionally return related resources, if requested wi
+            "policy-kind":"opa",
+            "status-timestamps":{
+               "passed-at":"2022-09-16T01:40:30+00:00",
+               "queued-at":"2022-09-16T01:40:04+00:00",
+               "running-at":"2022-09-16T01:40:08+00:00"
+            },
+            "created-at":"2022-09-16T01:39:07.782Z",
+            "updated-at":"2022-09-16T01:40:30.010Z"
+         },
+         "relationships":{
+            "policy-attachable":{
+               "data":{
+                  "id":"ts-yxskot8Gz5yHa38W",
+                  "type":"task-stages"
+               }
+            },
+            "policy-set-outcomes":{
+               "links":{
+                  "related":"/api/v2/policy-evaluations/poleval-8Jj9Hfoz892D9WMX/policy-set-outcomes"
+               }
+            }
+         },
+         "links":{
+            "self":"/api/v2/policy-evaluations/poleval-8Jj9Hfoz892D9WMX"
+         }
+      }
+   ]
+}
+```
+
+
+## List the Policy Outcomes of the Policy Evaluation
+
+This endpoint is only available for OPA policies.
+
+`GET /policy-evaluations/:policy_evaluation_id/policy-set-outcomes`
+
+| Parameter                   | Description                                                |
+| --------------------------- | ---------------------------------------------------------- |
+| `:policy_evaluation_id`     | The ID of the policy evaluation the outcome belongs to get |
+
+This endpoint allows you to list policy set outcomes that are part of the policy evaluation.
+
+| Status  | Response                                      | Reason                                                  |
+| ------- | --------------------------------------------- | ------------------------------------------------------- |
+| [200][] | [JSON API document][]                         | Success                                                 |
+| [404][] | [JSON API error object][]                     | Policy evaluation not found                             |
+
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling does not automatically encode URLs.
+
+| Parameter                          | Description                                                             |
+|----------------------------------- | ----------------------------------------------------------------------- |
+| `page[number]`                     | **Optional.** If omitted, the endpoint returns the first page.          |
+| `page[size]`                       | **Optional.** If omitted, the endpoint returns 20 agent pools per page. |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/policy-evaluations/pol-o7tAtsCxnGhPHCfv/policy-set-outcomes
+```
+
+### Sample Response
+
+```json
+{
+   "data":[
```

Comment:
Indentation seems off throughout the sample responses. Just go through and confirm the whitespace throughout.

```suggestion
   "data": [
```